### PR TITLE
Added user table.

### PR DESCRIPTION
**Pull request by mfduar8766**
@smuntean  @iangornall

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -8,4 +8,10 @@ CREATE TABLE decoy_colors (
   id SERIAL PRIMARY KEY,
   level VARCHAR(255) NOT NULL,
   decoy_color VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  email VARCHAR(50) UNIQUE,
+  user_password VARCHAR(10) UNIQUE
```
Comment (sarathrasher) [inline schema.sql]: Password should not be unique

File: schema.sql [hunk 2]
```diff
@@ -8,4 +8,10 @@ CREATE TABLE decoy_colors (
   id SERIAL PRIMARY KEY,
   level VARCHAR(255) NOT NULL,
   decoy_color VARCHAR(255) NOT NULL
+);
+
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  email VARCHAR(50) UNIQUE,
```
Comment (sarathrasher) [inline schema.sql]: This should be VARCHAR(255)

File: schema.sql (hunk 1 above)
Comment (sarathrasher) [reply schema.sql]: Also let's just make this a standard VARCHAR(255)

**Review by mfduar8766 — COMMENTED** [review]
added 255 for characters length

**Review by sarathrasher — APPROVED** [review]
👍

**Pull request merged**